packages/edge/src/__tests__/lib/commands/channels-util.test.ts

```diff
@@ -146,6 +151,63 @@ describe('channels-util', () => {
 			expect(listChannelsMock).toHaveBeenCalledTimes(1)
 			expect(listChannelsMock).toHaveBeenCalledWith({ includeReadOnly: true })
 		})
+
+		describe('defaultConfig', () => {
+			test('getItem uses channels.get', async () => {
+				chooseChannelOptionsWithDefaultsSpy.mockReturnValueOnce(
+					{ allowIndex: false, useConfigDefault: true } as ChooseChannelOptions)
+				selectFromListMock.mockImplementation(async () => 'chosen-channel-id')
+
+				expect(await chooseChannel(command, 'prompt message', undefined,
+					{ useConfigDefault: true })).toBe('chosen-channel-id')
+
+				const defaultValue = selectFromListMock.mock.calls[0][2].defaultValue
+
+				expect(defaultValue).toBeDefined()
+				const getItem = defaultValue?.getItem as (id: string) => Promise<Channel>
+				expect(getItem).toBeDefined()
+				getChannelsMock.mockResolvedValueOnce(channel)
+
+				expect(await getItem('id-to-check')).toBe(channel)
+
+				expect(getChannelsMock).toHaveBeenCalledTimes(1)
+				expect(getChannelsMock).toHaveBeenCalledWith('id-to-check')
+			})
+
+			test('userMessage returns expected message', async () => {
+				chooseChannelOptionsWithDefaultsSpy.mockReturnValueOnce(
+					{ allowIndex: false, useConfigDefault: true } as ChooseChannelOptions)
+				selectFromListMock.mockImplementation(async () => 'chosen-channel-id')
+
+				expect(await chooseChannel(command, 'prompt message', undefined,
+					{ useConfigDefault: true })).toBe('chosen-channel-id')
+
+				const defaultValue = selectFromListMock.mock.calls[0][2].defaultValue
+
+				expect(defaultValue).toBeDefined()
+				const userMessage = defaultValue?.userMessage as (channel: Channel) => string
+				expect(userMessage).toBeDefined()
+
+				expect(userMessage(channel)).toBe('using previously specified default channel named "channel name" (channel-id)')
+			})
+		})
+
+		test('defaultConfig.userMessage return valid message', async () => {
```

Comment:
Is this a duplicate of the previous test?

Reply:
Hmm, not sure how I managed that. Removed now.